src/components/Feeds/AllyFeed.js

```diff
@@ -62,6 +63,7 @@ const AllyFeed = props => {
       <Box
         maxW={theme.containers.main}
         paddingX={[null, theme.spacing.base, theme.spacing.lg]}
+        marginBottom="3.688rem"
```

Comment:
@juanri0s this is such a good change.  I noticed this before and was starting to get palpitations about it... and then I forgot! 

With that said - I feel like it might be a good idea to use `theme.spacing.lg` (which is `3rem`) here to keep consistent with the site theme:

```suggestion
        marginBottom={theme.spacing.lg}
```

Reply:
Yupp I'll take your suggestion! I just looked at the figma file and took the margin between the card and 'pagination'.